api/client/service/opts.go

```diff
@@ -253,6 +307,21 @@ func (m *MountOpt) Value() []swarm.Mount {
 	return m.values
 }
 
+func getMountType(fields []string) (swarm.MountType, []string) {
```

Comment:
This was the kind of mistake made in original grammar. Let's not make the same mistakes.